src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AgeGroup
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Intervention
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.InterventionType
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class InterventionDTO(
+  val id: UUID? = null,
+  val createdAt: OffsetDateTime? = null,
+  val title: String,
+  val description: String,
+  val ageGroup: AgeGroup,
+  var type: InterventionType,
+  var serviceCategory: String,
+  var serviceProvider: String,
```

Comment:
i think it would make sense to have a service provider DTO which potentially includes an id that can be used by the front end to get more information about the SP.

Reply:
yeah - I wasn't sure about this tbh...but left as a string as I wasn't sure if it would be needed. I'll add that in if we're gonna need it.